src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss

```diff
@@ -0,0 +1,83 @@
+@import "mixins";
+@import "variables";
+
+$row-height: 70px;
+
+:host {
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    margin-bottom: $basic-padding;
+    padding: 0 $basic-padding * 2;
+    max-height: $row-height;
+}
+
+.dot-add-rows__container {
```

Comment:
You don't need to nest this classes, another of the benefits of use BEM is that the specifity comes "for free" because the name of the classes, it's kind of implicit.